schema/gene.js

```diff
@@ -67,14 +68,18 @@ const Gene = {
       type: new GraphQLNonNull(GraphQLString),
     },
   },
-  resolve: (root, { id }, request, { fieldASTs }) => {
+  resolve: (root, { id }, request, other) => {
```

Comment:
any idea what this other field should be called, I seem to be getting it as null on relay update-y requests